components/Tooltip/Tooltip.jsx

```diff
@@ -117,6 +122,7 @@ Tooltip.propTypes = {
 Tooltip.defaultProps = {
   placement: 'top',
   visible: false,
+  multiline: false,
```

Comment:
How about of create a new example, using this new prop, in story of storybook, for documentation?